client-js/app/index.html

```diff
@@ -33,6 +33,7 @@
   <link rel="stylesheet" href="/styles/main.css">
   <script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="import" href="/elements/elements.html">
+  <!-- PROD_MODE -->
```

Comment:
It's just a marker so the dev mode code knows where to insert its script tag. I'm happy to change this to something more meaningful. Originally I wrote PROD_MODE thinking that we wouldn't want something like DEV_MODE_SPECIAL_CODE_GOES_HERE in the html when we do serve the file in prod, but in fact the `html` gulp task strips all the comments anyway, so we can put whatever we want.